mobile/library/cc/engine_builder.cc

```diff
@@ -245,6 +245,11 @@ EngineBuilder& EngineBuilder::setSocketReceiveBufferSize(int32_t size) {
   return *this;
 }
 
+EngineBuilder& EngineBuilder::setUdpSocketSendBufferSize(int32_t size) {
```

Comment:
do we even need this function?
if we keep it let's test it but I'd be inclined to either set the default correctly, or wire up all the APIs so we can do experiments.

Reply:
It makes it easier to experiment with different send / buffer sizes in g3 without having to do upstream changes. We have the tests for this in `envoy_config_test.cc`.